### ISSUE TITLE
Migrate from Google Translate Toolkit

### ISSUE THREAD

**Issue by mdoelker** [trufi-association/trufi-core#330]
Google Translate Toolkit is shutting down and it is the only platform supporting dart intl's ARB format. Track progress in this issue https://github.com/dart-lang/intl/issues/230 and come up with a migration plan to a different platform.

This might help https://github.com/bmw-tech/arb-converter-cli

**Comment by christophhanser** [comment]
Sorry for asking maybe stupid but would Localization Lab help?

**Comment by mdoelker** [comment]
Published tool to convert to XLIFF and back https://www.npmjs.com/package/arb-convert

**Comment by mdoelker** [comment]
Working on extending our bash scripts to automate import/export from our new localization provider.

**Comment by md-weber** [comment]
With Flutter 2.0 we changed the way internationalization works entirely in the Trufi Core. Now we are using the default way gen-l10n that got introduced by Flutter 1.22. 

I created a Script to get the translations directly from Lokalise via the [Lokalise2 CLI](https://github.com/lokalise/lokalise-cli-2-go). In the future this will be a one-way process, we will add translated Strings to our Lokalise project and can execute the script to receive the translations in our project.

I created more information in our Wiki. 
- [How to work with custom Translations](https://github.com/trufi-association/trufi-core/wiki/Custom-Translations)
- [How to update the strings from Lokalise](https://github.com/trufi-association/trufi-core/wiki/Translations-Update-Guide)
